examples/jax/debugging/simple_spread/feedforward/decentralised/run_mappo.py

```diff
@@ -101,7 +101,8 @@ def network_factory(*args: Any, **kwargs: Any) -> Any:
         num_epochs=15,
         num_executors=1,
         multi_process=True,
-        terminal="current_terminal",
+        parameter_termination_key="executor_steps",
+        parameter_termination_value=1500,
```

Comment:
I don't see where these values are actually being set in MAPPO. Seems to be using the default `config` values. Also, personal preference, but I would rather these be passed through as a tuple to denote that they have to be supplied together.

Reply:
Agreed on the tuple or dict and I think also remove the parameter from the variable name, I know we discussed adding it yesterday, but to an end user `parameter_termination_condition` would be more confusing. So maybe something like `termination_condition`

Reply:
Changed to take in a `termination_condition` dictionary instead.